packages/aws-cdk/lib/docker.ts

```diff
@@ -44,8 +51,10 @@ export async function prepareContainerAsset(assemblyDir: string,
   debug(' 👑  Preparing Docker image asset:', contextPath);
 
   try {
-    const ecr = await toolkitInfo.prepareEcrRepository(asset);
-    const latest = `${ecr.repositoryUri}:latest`;
+    const repositoryName = asset.repositoryName ?? 'cdk/' + asset.id.replace(/[:/]/g, '-').toLowerCase();
+    const ecr = await toolkitInfo.prepareEcrRepository(repositoryName);
+    const imageTag = asset.imageTag ?? 'latest';
```

Comment:
Can `imageTag` ever be set here?

Reply:
Theoretically it can because the protocol supports it (and I have a test that verifies it) but practically after 1.22.0 the app will never use it.